src/AuditTrailModule/javasource/test_crm/tests/expected/ExpectedLog.java

```diff
@@ -0,0 +1,93 @@
+package test_crm.tests.expected;
+
+import static org.junit.Assert.*;
+
+import java.util.Date;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+import com.mendix.core.CoreException;
+import com.mendix.systemwideinterfaces.core.IMendixIdentifier;
+
+import audittrail.proxies.Log;
+import audittrail.proxies.TypeOfLog;
+import test_crm.tests.actual.ActualLog;
+import test_crm.tests.actual.ActualLogLine;
+
+public class ExpectedLog {
+	final Integer numberOfChangedMembers;
+	final TypeOfLog typeOfLog;
+	final String objectType;
+	final IMendixIdentifier logUser;
+	final Date afterDate;
+	final String superClassName;
```

Comment:
It's the id of the object to which this log refers to. We use it to fetch the log object itself, so in a way we are already validating it.

The only validation we're not making yet is that no log entries are generated with the incorrect reference id, but that would be of limited value, imo. I can use the association from object to the log entry itself, but I cannot do so for the delete case (which is why Log has its own reference as well, so that deleted objects still have a log entry).

I'll see if I can extend this after the other fixes are done.